workspace/utilities/lib/fx-url-creator.xsl

```diff
@@ -76,127 +135,115 @@
 						
 						<xsl:call-template name="framework-288-url-creator-page-route">
 							<xsl:with-param name="item" select="$item" />
+							<xsl:with-param name="forceAllowArgs" select="$forceAllowArgs" />
 						</xsl:call-template>
 					</xsl:for-each>
 					
 					<!-- Index page -->
-					<xsl:if test="@id = '1'">
+					<xsl:if test="@id = '1' and $forceAllowArgs = false()">
```

Comment:
Why ?

Reply:
The index can have `/*` now for the _pages_ *section* in the cms

Reply:
I do not like 'force' parameters. They smell.

How can we make the template to have a single 'allow args' parameter ?

Reply:
`f288-model-hello` and `f288-allo-args` is still supported by this change ;)

Reply:
Yeah ok, but I do not like having a 'allowArgs' and 'forceAllowArgs' variable. There should be only one.